tests/standalone_tests/test_post_algorithms.py

```diff
@@ -87,8 +87,21 @@ def get_parametrization_list_success_cases():
     "algorithm_name, request_dict, expected_response",
     get_parametrization_list_success_cases(),
 )
-def test_post_testing_algorithms(algorithm_name, request_dict, expected_response):
-    algorithm_url = algorithms_url + "/" + algorithm_name
+def test_post_testing_algorithms(
```

Comment:
I think that this test is not needed at all, let's discuss it tomorrow.

The reason I want to remove this test, if possible, is that it uses the tmp node. This should be avoided since it takes time to setup the node.